platform/lib/platform_web/live/media_live/show.html.heex

```diff
@@ -23,37 +23,42 @@
         <%= if(
        Permissions.can_edit_media?(@current_user, @media, Attribute.get_attribute(:description))
       ) do %>
-          <%= live_patch(class: title_classes <> "hover:bg-neutral-200 focus:bg-neutral-200",
-              to: Routes.media_show_path(@socket, :edit, @media.slug, :description),
-              "data-tooltip": "Edit description",
-              replace: true
-            ) do %>
+          <.link
+            class={title_classes <> "hover:bg-neutral-200 focus:bg-neutral-200"}
```

Comment:
```suggestion
            class={title_classes <> " hover:bg-neutral-200 focus:bg-neutral-200"}
```

Not your error, but re-reading this code I spot an error I introduced awhile ago :)